File: C++/Ram.cpp
#include <bits/stdc++.h>
using namespace std;

class Ram
// kelas yang digunakan untuk mengimplementasikan Ram
{
private:
    // private atribute dari kelas Ram
    string capacity;
    int price;

public:
    // konstruktor
    Ram() {}
    Ram(string capacity, int price)
    {
        this->capacity = capacity;
        this->price = price;
    }

    // mengeset nilai atribut capacity
    void setCapacity(string capacity)
    {
        this->capacity = capacity;
    }

    // mengembalikan nilai atribut capacity
    string getCapacity()
    {
        return this->capacity;
    }

    // mengeset nilai atribut price
    void setPrice(int price)
    {
        this->price = price;
    }

    // mengembalikan nilai atribut price
    int getPrice()
    {
        return this->price;
    }

    // menampilkan atribut Ram
    void printRam()
    {
        cout << "Ram Capacity : " << this->getCapacity() << endl;
        cout << "Ram Price : " << this->getPrice() << endl;
    }

    // destructor
    ~Ram() {}
};

File: Python/Ram.py
# kelas yang digunakan untuk mengimplementasikan Ram
class Ram:
    # private atribute dari kelas Ram
    capacity = ""
    price = 0

    # konstruktor
    def __init__(self, capacity="", price=0):
        self.__capacity = capacity
        self.__price = price

    # mengeset nilai atribut capacity
    def setCapacity(self, capacity):
        self.__capacity = capacity

    # mengembalikan nilai atribut capacity
    def getCapacity(self):
        return self.__capacity

    # mengeset nilai atribut price
    def setPrice(self, price):
        self.__price = price

    # mengembalikan nilai atribut price
    def getPrice(self):
        return self.__price

    # menampilkan atribut Ram
    def printRam(self):
        print("Ram Capacity : " + self.getCapacity())
        print("Ram Price : " + str(self.getPrice()))
